lib/src/model/old_product_result.g.dart

```diff
@@ -1,9 +1,9 @@
 // GENERATED CODE - DO NOT MODIFY BY HAND
 
-// ignore_for_file: deprecated_member_use_from_same_package
-
 part of 'old_product_result.dart';
 
+// ignore_for_file: deprecated_member_use_from_same_package
```

Comment:
I'm not (100%) a maniac, but please put the "ignore" comment where it was.
Much better later when looking at the code we'll see that no one touched this file for months, instead of a recent useless change.